stack_examples/aws_ssm_package_installer/default.tfvars

```diff
@@ -0,0 +1,5 @@
+region_resource = ""
```

Comment:
If you look at other example, I think they try either give real values here or an idea what real values would be via comments.